stubs/check.stub

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace {{ namespace }};
+
+use UKFast\HealthCheck\HealthCheck;
+
+class {{ class }} extends HealthCheck
+{
+    protected $name = '';
+
+
+    public function __construct()
+    {
+        //
+    }
+
+    public function status()
+    {
+        return $this->okay();
+    }
+}
```

Comment:
Would be good to have a new line here, it'd stop phpcs from flagging it

Reply:
I'll resolve this as I've added a suggestion for it below. 

https://github.com/ukfast/laravel-health-check/pull/30#discussion_r498067685